Create material_gl.arb

hesos opened the pull request:
Galician localizations

hesos committed:
Create material_gl.arb

dariotrombello commented:
Incomplete
Take a look on this example how to add a translation: https://github.com/flutter/flutter/commit/cef11417dae7fb9f42792a9926b546e385ee2204#diff-0b483aae4fc2e8b4df4dfd65486bfc03

dnfield committed:
Use mDNS to discover the device port (#26944)

* Discover port over mDNS

* opt in, only for iOS for now

goderbauer committed:
Typo fixes for notched_shapes.dart (#27097)

Commit:
Add flutter_build package for codegen and version sync with tool (#26978)

Commit:
Wrap dart:convert to track utf8 decode failures (#26650)

Commit:
Updated DropdownButton docs (#27042)

xster committed:
Fix gsutil.py call for windows (#27113)

Commit:
prevent _computeColumnWidths from getting stuck due to double precision (#27112)

engine-flutter-autoroll committed:
Roll engine 36acd02c9412..9b6d5031a3c0 (5 commits) (#27104)

* 4f02f7e22 Roll src/third_party/skia aa6ca0a8bfc1..74b6cf3a7215 (2 commits) (flutter/engine#7583)

* fe2609f5c Roll src/third_party/skia 74b6cf3a7215..a7cb690dc8fe (1 commits) (flutter/engine#7584)

* 204a09910 Roll src/third_party/skia a7cb690dc8fe..9bc76d96f986 (1 commits) (flutter/engine#7585)

* 5cfc0249a Roll src/third_party/skia 9bc76d96f986..d41c1876d834 (6 commits) (flutter/engine#7586)

* 9b6d5031a Roll src/third_party/dart a5030ed92f..a5030ed92f (0 commits)

engine-flutter-autoroll committed:
Roll engine 36acd02c9412..9b6d5031a3c0 (5 commits) (#27135)


https://github.com/flutter/engine/compare/36acd02c9412...9b6d5031a3c0

git log 36acd02c9412d0c1e6acbe0fbcb3dd03cda038df..9b6d5031a3c0a4a6b1b5bd66a8e6a7c3c2c7ea0b --no-merges --oneline
9b6d5031a Roll src/third_party/dart a5030ed92f..a5030ed92f (0 commits)
5cfc0249a Roll src/third_party/skia 9bc76d96f986..d41c1876d834 (6 commits) (flutter/engine#7586)
204a09910 Roll src/third_party/skia a7cb690dc8fe..9bc76d96f986 (1 commits) (flutter/engine#7585)
fe2609f5c Roll src/third_party/skia 74b6cf3a7215..a7cb690dc8fe (1 commits) (flutter/engine#7584)
4f02f7e22 Roll src/third_party/skia aa6ca0a8bfc1..74b6cf3a7215 (2 commits) (flutter/engine#7583)

The AutoRoll server is located here: https://autoroll.skia.org/r/flutter-engine-flutter-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff (jsimmons@google.com), and stop
the roller if necessary.

dnfield committed:
Add docs and sample for takeException (#27140)

* Add docs and sample for takeException

* fix sample

DanTup committed:
Change "Starting Xcode build" text to "Running xcode build" (#27181)

This status shows for the duration of the run, not just while starting. It looks kinda weird in the editors to show a progress bar for "Starting Xcode build" and then removing it once it completes.

DanTup committed:
Improve unsupported text (#27186)

jimshepherd committed:
Fix DropDownButton with no items resulting in RenderFlex overflow (#26143)

dnfield committed:
Revert f9e6242db (#26944) (#27191)

* Revert f9e6242db

* fix pubspec

* finish pubspec upgrade

Commit:
PopupMenuDivider.represents() paramter must be void, not Null (#27195)

jslavitz committed:
Adds haptic vibration to Cupertino switch (#27114)

* adds haptic vibration to switch

jslavitz committed:
Adds a fade in and out, rounds corners, fixes offset and fixes height of cursor on iOS (#24876)

* Fixes cursor on iOS devices

chinmaygarde committed:
Add a flag to enable tracing to systrace. (#27208)

rrousselGit committed:
Make reassemble public (#26900)

mit-mit committed:
Wrap lines at 80 characters (#26285)

goderbauer committed:
Remove obsolete ignore: (#27199)

gspencergoog committed:
Implement hover support for mouse pointers. (#24830)

This implements support for hovering mouse pointers, so that mice connected to Android devices, and ChromeOS devices running Android apps will work properly.

It teaches flutter_test about hover events, which required changing how they are created and used.

Also modifies AnnotatedRegion to allow a region that can be located someplace other than just the origin.

Along with tests for all of the above.

Fixes #5504

scheglov committed:
Add default values for optional parameters. (#27197)

Commit:
Add module checking (#27122)

a14n committed:
format initializer list of constructors (#27111)

akindone committed:
fix: cuertino dialog action background blur effect (#25076)

Commit:
[cupertino_icons] add car, bus, train, paw, controller, and flask icons. (#26774)

* [cupertino_icons] add car, bus, train, paw, controller, and flask icons.

* [#26774] remove whitespace

a-siva committed:
Add support for experimental flags during hot reload. (#27252)

* Add support for experimental flags during hot reload.

kangwang1988 committed:
Let injectPlugins integrate custom pods (#26970)

goderbauer committed:
Update Align docs (#27260)

goderbauer committed:
Add arguments for pushing named routes (#27058)

engine-flutter-autoroll committed:
Roll engine 9b6d5031a3c0..006f41026c90 (40 commits) (#27269)


https://github.com/flutter/engine/compare/9b6d5031a3c0...006f41026c90

git log 9b6d5031a3c0a4a6b1b5bd66a8e6a7c3c2c7ea0b..006f41026c90d8c22a93e2580d6bf34a72ae7e0e --no-merges --oneline
006f41026 Revert buildtools roll back to bac220c (flutter/engine#7633)
a6687f12e Roll src/third_party/skia 8f388ceadd79..3f42e98f461a (3 commits) (flutter/engine#7632)
5cd1d4e57 Roll src/third_party/dart 05a083a95a..011a1239fc (5 commits) 011a1239fc Support for out-of-band worker in AnalysisDriverScheduler. b625926038 [ VM / dart:io ] Updated `Link` implementation for Windows to use actual symbolic links. 09e6a689c2 parse nested control flow collection entries 213cfdc85e Update AstBuilder to generate new for statement structure 93a56336a3 Parse simple for control flow collection entires
474d8837a Rename FlutterResult in embedder.h (flutter/engine#7567)
17626006b Roll src/third_party/dart 219a2fcf66..05a083a95a (12 commits) 05a083a95a [vm] Update status file for entrypoints_verification_test. 2bb38e62e8 [infra] Use fewer cores on Android shards 267e91a0ea Re-land &#34;[vm] Prevent access to non-annotated members through native API.&#34; d73d3183a2 Fix dead_code hint 24fa01dbae [release] Update changelog with changes from dev branch f5fd281150 Prevent unsound promoted bounds from being used as arguments cf56801308 Garbage-collect compiled libraries 8fc9eaf06c Support to run co19_2 on ddk 812a6af41a Anonymous mixin applications are abstract 977e5b31e2 Support type variables on function types and intersection types f822d0abb8 Make CoreType test friendly d025ff5367 Copy subtype tests
20d8388b5 Check in GEM_HOME for jazzy (flutter/engine#7628)
d79f421a6 Roll src/third_party/skia 41169206c475..8f388ceadd79 (6 commits) (flutter/engine#7629)
78e145d05 Improve PathMetrics (flutter/engine#7621)
18f014590 Update buildtools to c9e5400c9e03a0cfb7313d14fde38525399a7715 (flutter/engine#7602)
57b0afaf3 Roll src/third_party/skia a09731e5cf3e..41169206c475 (4 commits) (flutter/engine#7626)
add0507d6 Roll src/third_party/dart 601094e3f2..219a2fcf66 (6 commits) 219a2fcf66 Revisit existing constants in the constants transformer. 97cb74cf63 Tweak the subtype rule for Null 5fbf15d9c4 Mixins have no static members d5a558d6c4 [VM/AOT] Make DartTemplateCall::Selector() work even if there is no ICData c5a954794b Fix PackageBuildWorkspacePackage&#39;s contains d84851daf8 [vm/bytecode] Capture receiver if creating a closure
e2d2fd496 Roll src/third_party/skia be6549a59d50..a09731e5cf3e (1 commits) (flutter/engine#7624)
56fe87dca Update license to sync with flutter/flutter (flutter/engine#7611)
68a0f9362 Roll src/third_party/skia 70041cf243b8..be6549a59d50 (1 commits) (flutter/engine#7623)
ec627aa17 Roll src/third_party/skia b13f0f5ef003..70041cf243b8 (2 commits) (flutter/engine#7620)
d6bb302af Roll src/third_party/skia 6c621f57ba53..b13f0f5ef003 (4 commits) (flutter/engine#7619)
51e879d6d Roll src/third_party/dart ec5ba8a252..601094e3f2 (7 commits) 601094e3f2 [vm/bytecode] Fix max used block id when compiling from bytecode ff2e8eb681 [vm, gc] Fix backwards write-barrier logic used by Object::Clone. 1ba59eb13d Make ERROR the default severity for StaticWarningCode. Remove _StrongModeTypeErrorProcessor. fc417a73ed [vm/compiler] Disable max block id check (for now) e02399da33 [vm/compiler] Introduce graph checker 2881a92c4e [vm/compiler] Fix missing incremental DOM update 95b017a888 Implement changeFile() and recompute available libraries and declarations.
79dc315dc Allow the engine to redirect traces to systrace via settings. (flutter/engine#7617)
28b27d4f1 Roll src/third_party/skia 8c0a1cad37fa..6c621f57ba53 (6 commits) (flutter/engine#7615)
956ecc947 Roll src/third_party/dart e169134203..ec5ba8a252 (6 commits) ec5ba8a252 Revert &#34;Issue 34437. Restore checking that optional parameters in derived classes should have the same default values as overridden.&#34; 7daa9f65fc Start supporting resolution of the code-as-ui constructs 69af7af70d Additional tests for interpolation and CONVERT_TO_MULTILINE_STRING. 5cf4e543b8 Handle mis-included parts differently f066c05319 Rename klass getter on InstanceConstant to classNode 42be4083bd Reproduction of dartbug.com/35715
f12e3994b Roll src/third_party/skia ede88ab2b414..8c0a1cad37fa (10 commits) (flutter/engine#7613)
74a8f1ff2 Roll src/third_party/skia 04c1cccaa45e..ede88ab2b414 (9 commits) (flutter/engine#7612)
787b93f41 Roll src/third_party/dart d927333c52..e169134203 (2 commits) e169134203 [Fasta] Fix a typo in a class name 591df5263f Fix typo in omitPlatform documentation
b83460888 Roll src/third_party/skia 04ac845bf755..04c1cccaa45e (1 commits) (flutter/engine#7608)
18a1f8572 Roll src/third_party/dart d0f92e8cc5..d927333c52 (6 commits) d927333c52 Add all SDK libraries, and context files belonging to the package / potions of it. f92b816268 Add &#39;Convert to multiline string&#39; quick assist. c5cfdc5279 Expose data member &#39;port&#39; for MessageEvents and automatically pull in markupsafe for go.sh script. 6699384ddd Issue 34437. Restore checking that optional parameters in derived classes should have the same default values as overridden. df34f65d90 Restore TLABs. ece54f3c43 [dart2js] Don&#39;t inline direct recursion
225a963ff Roll src/third_party/skia 80f4adf98fef..04ac845bf755 (2 commits) (flutter/engine#7606)
a472af71f Roll src/third_party/skia 27e5125dbc05..80f4adf98fef (1 commits) (flutter/engine#7605)
71e1b9530 Roll src/third_party/skia 02edccb02451..27e5125dbc05 (1 commits) (flutter/engine#7604)
0cbc2de43 Roll src/third_party/skia a5d41af0e8ae..02edccb02451 (1 commits) (flutter/engine#7603)
0b2248a87 Roll src/third_party/skia e695c1855390..a5d41af0e8ae (1 commits) (flutter/engine#7601)
9c7cb26e0 Roll src/third_party/skia 6535d1f283e2..e695c1855390 (1 commits) (flutter/engine#7600)
e172daef7 Roll src/third_party/skia d3a6a8162cfa..6535d1f283e2 (1 commits) (flutter/engine#7599)
37c84f196 Roll src/third_party/skia 9d0568939a9f..d3a6a8162cfa (1 commits) (flutter/engine#7598)
9557311b8 Roll src/third_party/skia 4d29e6d7432f..9d0568939a9f (1 commits) (flutter/engine#7597)
f50a6beca Roll src/third_party/dart c167cd3572..d0f92e8cc5 (7 commits) d0f92e8cc5 Remove StrongTypeSystemImpl. 2f3ebf70f6 Update definite assignment for the code-as-ui changes fe285e3e60 Update flow analysis for the code-as-ui changes 8e0bc806b9 bump to linter 0.1.79 c7f6c86128 Fix registration of (un)provided parameters. 6fae43b8af Check exact type for set literals (issue 35742) eadf9679f3 Fix #35750 by passing a real ContextBuilder to createWorkspace.
ffc56bc03 Roll src/third_party/skia 5d982feca871..4d29e6d7432f (6 commits) (flutter/engine#7595)
6cf1a0766 Roll src/third_party/dart bb4a880777..c167cd3572 (9 commits) c167cd3572 [VM] Experimental flags in dartkp tests 759234b05a [vm] Fix disassembler test after f496e538f4587f29562e161ca6b06f169396f499 bb560bb233 [ VM / dart:io ] Adding to a closed IOSink now throws a StateError 983b77dda7 Remove unused argument allowNullableDereference f8fef94243 Authomatically add dependencies from pubspec.yaml files. 169331abb1 [vm] Misc timeline tweaks. 874f9d0bdf Refactor for statement parsing 9043e5be44 [vm, gc] Remove --gc_at_alloc. 27760001f5 [vm] Fix build after f496e538f4587f29562e161ca6b06f169396f499
08f3c257f Roll buildroot to b4d21cb2a64d63218c8d99533d9c14e99201e8d8 (flutter/engine#7591)
0b1722dd8 Roll src/third_party/skia d41c1876d834..5d982feca871 (5 commits) (flutter/engine#7590)
686a0a094 Roll src/third_party/dart a5030ed92f..bb4a880777 (2 commits) bb4a880777 Report duplicate set entries with a different error f496e538f4 [vm] Decouple assemblers from runtime.

The AutoRoll server is located here: https://autoroll.skia.org/r/flutter-engine-flutter-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md
...

goderbauer committed:
Add missing comma to fix build (#27274)

engine-flutter-autoroll committed:
Roll engine 006f41026c90..01b4f4e28065 (2 commits) (#27289)


https://github.com/flutter/engine/compare/006f41026c90...01b4f4e28065

git log 006f41026c90d8c22a93e2580d6bf34a72ae7e0e..01b4f4e280659a7333a77a570ff32122a2d947ad --no-merges --oneline
01b4f4e28 Roll src/third_party/dart 011a1239fc..2632e201b7 (10 commits) 2632e201b7 Support for DeclaredIdentifier(s) in new ForEachPartsWithDeclaration/ForElement(s). 689b17d69b [vm/compiler] Fix missing incremental DOM update b5d9e66963 Add extra polyfill for .nonce in our deferred loading logic to support FF 1e92b5ebd0 [vm] Note the flow graph builder when checking type feedback compatibility. 7ee0e00f0b [vm, gc] Reset growth policy after finishing kernel loading. 4e0aa3ff37 Run ddb with debug port open b196cd2acb Issue 34437. Restore checking that optional parameters in derived classes should have the same default values as overridden. 25d8456c48 Small improvements for DeclarationsTracker. 081160036a Fix #35306, improper recursion detection. 6f5556c88a Prepare to publish analyzer version 0.35.0.
1624619ff Roll src/third_party/skia 3f42e98f461a..c749f3bfd0e3 (5 commits) (flutter/engine#7636)

The AutoRoll server is located here: https://autoroll.skia.org/r/flutter-engine-flutter-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff (dnfield@google.com), and stop
the roller if necessary.

scheglov committed:
Update additionalTime in TestWidgetsFlutterBinding.runAsync() to 1000. (#27256)

Commit:
add support to multiroot scheme to PackageUriMapper (#27207)

tvolkert committed:
Allow bundle identifier to be surrounded with quotes. (#26300)

Picked up from #22498, originally by @jugyo

msal4 committed:
fix #26207 .gitignore only include project root build directory (#26388)

sjindel-google committed:
Pass --verify-entry-points in debug mode. (#27032)

jonnyandrew committed:
Include empty config in 'flutter config' output (#25094)

Commit:
Inject KernelCompiler via KernelCompilerFactory (#27211)

goderbauer committed:
Fix asserts for initialDateTime in CupertinoDatePicker (#27272)

goderbauer committed:
Remove all obsolete "// ignore:" (#27271)

kangwang1988 committed:
Add2App: Fix crash resulted from hard-code module 'app' based resource copy. (#27154)

jason-simmons committed:
Declare a system message channel for Skia configuration (#26262)

Fixes https://github.com/flutter/flutter/issues/25244

engine-flutter-autoroll committed:
Roll engine 01b4f4e28065..31a7f4d5205d (2 commits) (#27301)


https://github.com/flutter/engine/compare/01b4f4e28065...31a7f4d5205d

git log 01b4f4e280659a7333a77a570ff32122a2d947ad..31a7f4d5205df9d73fcdc148940d65d0efe55d95 --no-merges --oneline
31a7f4d52 Roll src/third_party/dart 2632e201b7..71bee8f05e (1 commits) 71bee8f05e Fix serializing procdure when realClass is Object from dart:core.
ed3ed9bc9 Roll src/third_party/skia c749f3bfd0e3..673a048b209c (1 commits) (flutter/engine#7639)

The AutoRoll server is located here: https://autoroll.skia.org/r/flutter-engine-flutter-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff (dnfield@google.com), and stop
the roller if necessary.

Commit:
handle missing curl (#27295)

rami-a committed:
Add Material/Card borderOnForeground flag to allow border to be painted behind the child widget (#27297)

In certain situations, a developer may require the border of a Material to be painted behind its child. For example a Card widget that has a full width image across the top half. In that scenario, the image should ideally be painted above the border with regards to z-position.

This change exposes a flag on Material widget to achieve this behavior. Additionally, the same flag is exposed on Card widget to allow the Card widget to pass this down to its Material.

I added a couple golden tests to verify this new behavior. Goldens are here:
https://github.com/flutter/goldens/commit/46a3d26acbb1b0d72b6b02c30f03b9dbda7d5bdf

zanderso committed:
[flutter_tool,doctor] Fix and test gen_snapshot failure message (#27253)

engine-flutter-autoroll committed:
15f2b92cc Roll src/third_party/skia 673a048b209c..70ebd9ca0616 (8 commits) (flutter/engine#7641) (#27313)


https://github.com/flutter/engine/compare/31a7f4d5205d...15f2b92cce91

git log 31a7f4d5205df9d73fcdc148940d65d0efe55d95..15f2b92cce916982b7dd8ce658bbf2a465c06ba4 --no-merges --oneline
15f2b92cc Roll src/third_party/skia 673a048b209c..70ebd9ca0616 (8 commits) (flutter/engine#7641)

The AutoRoll server is located here: https://autoroll.skia.org/r/flutter-engine-flutter-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff (dnfield@google.com), and stop
the roller if necessary.

hesos committed:
Update date_localizations.dart

hesos committed:
Update localizations.dart

hesos committed:
Merge pull request #2 from hesos/patch-4

Update localizations.dart

hesos committed:
Merge pull request #3 from hesos/patch-3

Patch 3
